features/fixtures/ios/iOSTestApp/Info.plist

```diff
@@ -48,19 +50,19 @@
 	</array>
 	<key>bugsnag</key>
 	<dict>
+		<key>apiKey</key>
+		<string>0192837465afbecd0192837465afbecd</string>
```

Comment:
Do the e2e tests need an API key? I thought they were talking to a mock server?

Reply:
It's a good question.... There are various steps that expect this specific apiKey in headers etc, so Nick is following the existing pattern.  It's just curious that an obviously fake apiKey wasn't chosen in the first place.

Reply:
OK, as long as it's supposed to be there.

Reply:
I haven't added anything here!

Xcode decided to re-order some of the lines in the Info.plist when a new field (`UIFileSharingEnabled`) was added.